events-api/src/modules/tickets/tickets.controller.ts

```diff
@@ -0,0 +1,41 @@
+import { Body, Controller, Get, Param, Post, Put, Request, UseGuards } from '@nestjs/common';
+import { TicketsService } from './tickets.service';
+import { AuthTokenGuard } from 'src/shared/guards/auth-token.guard';
+import { CreateTicketReqDto } from './dto/requests/create-ticket.dto';
+import { PaymentDto } from './dto/requests';
+
+@Controller('tickets')
```

Comment:
If any manipulation with this controller requires a guard, you can use guards on the controller level.

Reply:
moved to controller level